### PR TITLE
echo-checkup: Add production manifest

### PR DESCRIPTION
**Pull request by orelmisan**
This adds a manifest to configure and run the echo-checkup.

Note: The following instructions assume the framework is installed.

To run the echo checkup:
```bash
$ kubectl apply -f checkups/echo/manifests/echo-checkup.yaml
```

To view the results:
```bash
$ kubectl get configmap echo-checkup-example-config -n kiagnose -o yaml
```
To delete all objects created by this manifest:
```bash
$ kubectl delete -f checkups/echo/manifests/echo-checkup.yaml
```

This PR depends on PR #36.

Signed-off-by: Orel Misan <omisan@redhat.com>

### CODE REVIEW

File: checkups/echo/manifests/echo-checkup.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: echo-checkup-example-config
```
Comment (ormergi) [inline checkups/echo/manifests/echo-checkup.yaml:5]: nit: No need to mention example

**Review by ormergi — COMMENTED** [review]
Hi thanks for adding this manifest, 
Regrading the commit message no need to mention "production"

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kiagnose/compare/b0fd52fa8ea6015672d30ee574c0262c351f43f4..850755f219149c163d71ff0457eb30d5a6382965): Changed ConfigMap's name from `echo-checkup-example-config` to `echo-checkup-config`.

File: checkups/echo/manifests/echo-checkup.yaml (hunk 1 above)
Comment (orelmisan) [reply checkups/echo/manifests/echo-checkup.yaml]: Done.

**Comment by orelmisan** [comment]
> Hi thanks for adding this manifest, Regrading the commit message no need to mention "production"

The "production" part of the title is meant to differentiate this manifest that is used by a user, from the previous manifest file that is used for development purposes.

**Comment by EdDev** [comment]
Please rebase

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kiagnose/compare/850755f219149c163d71ff0457eb30d5a6382965..80a01e77d3f86135f95f30beb733c1e1bd071ba4): Rebase

**Review by EdDev — APPROVED** [review]
Very nice, thanks!

@ormergi please have a look.

**Review by ormergi — APPROVED** [review]
Thanks

**Pull request merged**